src/Component/DependencyInjection/src/Argument/ArgumentInterface.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @memberOf Jymfony.Component.DependencyInjection
+ */
+class ArgumentInterface {
+    /**
+     * @function
+     * @name ArgumentInterface#getValues
+     *
+     * @return array
+     */
```

Comment:
Please explicit methods in this interface.
In addition, the correct way to document an array is `{[type]}`, where `type` can be a base type, `*` or a class name.